clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1595,6 +1717,14 @@ std::unique_ptr<lto::LTO> createLTO(
   Conf.OptLevel = OptLevel[1] - '0';
   Conf.DefaultTriple = Triple.getTriple();
 
+  // We need to set up the backend to use thinLTO
+  // even if we don't actually use it, and there is no
+  // backend for the spir64 triple, so override it to
+  // the SPIR-V backlend.
+  // TODO: Remove once SYCL uses the SPIR-V backend.
```

Comment:
+1 on this. I also briefly tried to use backend for 'linking' LLVM IR files. I am planning to revisit that PR (https://github.com/intel/llvm/pull/13395) soon. 
Also, eventually, we will use SPIRV64 (instead of SPIR64) as the target triple for SYCL purposes.
This change is agreeable as a stop gap. I think.
Adding @Naghasan for comments.

Thanks

Reply:
Sounds good to me, the fact we can't get anywhere with the `spir` triples is a bit frustrating.